src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -187,7 +187,7 @@ private IAsyncCollector<IStorageQueueMessage> BuildFromQueueAttribute(QueueAttri
 
             internal IStorageQueue GetQueue(QueueAttribute attrResolved)
             {
-                var account = Task.Run(() => this._accountProvider.GetStorageAccountAsync(attrResolved, CancellationToken.None)).GetAwaiter().GetResult();
```

Comment:
While I agree that it might make sense for this to be a sync interface, it has been async from virtually the beginning of the SDK. I think we should track optimizing these Task.Runs away in a separate PR. It's not required for your fix, right?